frontend/src/views/Inbox.vue

```diff
@@ -99,27 +95,21 @@ export default {
       let count = state.unreadList.length;
       const inboxList = state.unreadList.map((item) => item);
 
-      inboxList.forEach((item: Inbox) => {
-        inboxStore
-          .patchInbox({
-            inboxId: item.id,
-            inboxPatch: {
-              status: "READ",
-            },
-          })
-          .then(() => {
-            const i = state.unreadList.findIndex(
-              (unreadItem) => unreadItem.id == item.id
-            );
-            if (i >= 0) {
-              state.unreadList.splice(i, 1);
-            }
-            count--;
-            if (count == 0) {
-              inboxStore.fetchInboxSummaryByUser(currentUser.value.id);
-            }
-            state.readList.push(item);
-          });
+      inboxList.forEach((item) => {
+        item.status = InboxMessage_Status.STATUS_READ;
+        inboxV1Store.patchInbox(item).then(() => {
+          const i = state.unreadList.findIndex(
+            (unreadItem) => unreadItem.name == item.name
+          );
+          if (i >= 0) {
+            state.unreadList.splice(i, 1);
+          }
+          count--;
+          if (count == 0) {
+            inboxV1Store.fetchInboxSummary();
+          }
+          state.readList.push(item);
+        });
```

Comment:
Not involved in this PR. But this implementation will create tons of requests (N = count of unread items).
I guess we could improve this later.

Reply:
Pagination & cache when necessary, otherwise seems we cannot avoid this if we don't compose data in the backend.

Reply:
Yeah, we need batchUpdate.

Reply:
My mistake..Yes the `fetchInboxSummary` here is strange. I just align with the old code, we should refactor this later.